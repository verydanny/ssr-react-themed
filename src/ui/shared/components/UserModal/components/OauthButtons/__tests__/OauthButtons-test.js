import React from 'react'
import { shallow } from 'enzyme'
import { keyMirror } from '@rentpath/react-ui-utils'
import OauthButtonsThemed from '../OauthButtons'

const OauthButtons = OauthButtonsThemed.WrappedComponent

const theme = keyMirror([
  'Oauth_Container',
  'Oauth_Facebook',
  'Oauth_Googleplus',
  'Oauth_SocialMediaIcons',
  'Oauth_FacebookButton',
  'Oauth_GoogleButton',
])

const props = {
  theme,
}

describe('OauthButtons', () => {
  describe('Oauth button render properly', () => {
    const wrapper = shallow(<OauthButtons {...props} />)

    it('Oauth buttons output matches the snapshot', () => {
      expect(wrapper.html()).toMatchSnapshot()
    })
  })
})
