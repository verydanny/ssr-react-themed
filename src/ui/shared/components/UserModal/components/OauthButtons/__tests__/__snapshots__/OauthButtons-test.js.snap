// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OauthButtons Oauth button render properly Oauth buttons output matches the snapshot 1`] = `

<div class="Oauth_Container">
  <a class="Oauth_FacebookButton Oauth_SocialMediaIcons"
     href="#"
  >
    <div style="height:50px"
         class="lazyload-placeholder"
    >
    </div>
    <p>
      Sign in with Facebook
    </p>
  </a>
  <a class="Oauth_GoogleButton Oauth_SocialMediaIcons"
     href="#"
  >
    <div style="height:50px"
         class="lazyload-placeholder"
    >
    </div>
    <p>
      Sign in with Google
    </p>
  </a>
</div>

`;
